src/Blocks/BlockPressurePlate.h

```diff
@@ -28,7 +28,26 @@ class cBlockPressurePlateHandler :
 			return false;
 		}
 
-		return (cBlockInfo::FullyOccupiesVoxel(a_Chunk.GetBlock(a_RelX, a_RelY - 1, a_RelZ)));
+		// TODO: check if the block is upside-down slab or upside-down stairs
+		BLOCKTYPE Block = a_Chunk.GetBlock(a_RelX, a_RelY - 1, a_RelZ);
```

Comment:
`a_RelY - 1` must be at least zero for this to work (Y coord ranges from 0 to 255). So when attempting to place a pressure plate at the bottom-most layer of the world, your code will read a block at Y coord `-1`; that is definitely not good and can crash the server.